.github/workflows/main/sconsOptions

```diff
@@ -46,6 +51,40 @@ DELIGHT_ROOT = os.environ["DELIGHT"]
 BUILD_DIR = os.environ["GAFFER_BUILD_DIR"]
 INSTALL_DIR = os.path.join( "install", os.environ["GAFFER_BUILD_NAME"] )
 
-PACKAGE_FILE = "%s.tar.gz" % os.environ["GAFFER_BUILD_NAME"]
+PACKAGE_FILE = "{}.{}".format(
+    os.environ["GAFFER_BUILD_NAME"],
+    {"win32" : "zip"}.get( sys.platform, "tar.gz" )
+)
 
 SPHINX = os.environ["GAFFER_SPHINX"]
+
+if sys.platform == "win32" :
+
+    # It would be nice to get this from the command line, but I don't know of any way
+    # to do that. It will need to be changed when upgrading Visual Studio versions.
+    toolSetVersion = "141"
+
+    # Get the Boost version from the include directory.
+    # It is named "boost-<majorVersion>_<minorVersion>"
+    boostInclude = glob.glob( os.path.join( os.environ["GAFFER_BUILD_DIR"], "include", "boost-*" ) )
+    if len( boostInclude ) > 0 :
+        boostVersion = os.path.split( boostInclude[0] )[-1].split( '-' )[1]
+
+        # Get the Python version, as in SConstruct but without the '.' separating major from minor version
+        pythonVersion = subprocess.check_output(
+            [ "python", "-c", "import sys; print( '{}{}'.format( *sys.version_info[:2] ) )" ],
+            universal_newlines=True
+        ).strip()
+
+        BOOST_LIB_SUFFIX = "-vc" + toolSetVersion + "-mt-x64-" + boostVersion
+        BOOST_PYTHON_LIB_SUFFIX = pythonVersion + "-vc" + toolSetVersion + "-mt-x64-" + boostVersion
+        LOCATE_DEPENDENCY_SYSTEMPATH = [
+            os.path.join( os.environ["GAFFER_BUILD_DIR"], "include", "boost-" + boostVersion )
+        ]
```

Comment:
Is it possible to change the Boost build config so it doesn't produce all these suffixes? Maybe not something to worry about now, but might be nice to look at when we're trying to unify the dependencies build process across platforms.

Reply:
Ahhhh, it is possible! `--release=system` when building boost is the magic word. Thanks for suggesting that, it's a very nice improvement. I will update the dependencies build and put out a new release to use when I submit the PR including building on Windows in Github workflows.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/4524/commits/e85beddd1edf45eb80f6380d939ee7dd2919560c